fineract-provider/src/main/resources/sql/migrations/core_db/V376__savings_last_transaction_date.sql

```diff
@@ -0,0 +1,24 @@
+--
```

Comment:
Please check if migration script number is still valid since there have been a couple of PRs merged since this PR was sent.